usaon_benefit_tool/__init__.py

```diff
@@ -88,49 +91,59 @@ def before_request():
-    from usaon_benefit_tool.routes.survey.relationships.data_product_application import (
-        data_product_application_bp,
-    )
-    from usaon_benefit_tool.routes.survey.relationships.observing_system_data_product import (
-        observing_system_data_product_bp,
-    )
-    from usaon_benefit_tool.routes.survey.sbas import societal_benefit_area_bp
-    from usaon_benefit_tool.routes.surveys import surveys_bp
+
+    # from usaon_benefit_tool.routes.survey import response_bp, survey_bp
+    # from usaon_benefit_tool.routes.survey.applications import application_bp
+    # from usaon_benefit_tool.routes.survey.observing_systems import observing_system_bp
+    # from usaon_benefit_tool.routes.survey.relationships.application_societal_benefit_area import (
+    #     application_societal_benefit_area_bp,
+    # )
+    # from usaon_benefit_tool.routes.survey.relationships.data_product_application import (
+    #     data_product_application_bp,
+    # )
+    # from usaon_benefit_tool.routes.survey.relationships.observing_system_data_product import (
+    #     observing_system_data_product_bp,
+    # )
+    # from usaon_benefit_tool.routes.survey.sbas import societal_benefit_area_bp
     from usaon_benefit_tool.routes.user import user_bp
     from usaon_benefit_tool.routes.users import users_bp
 
     app.register_blueprint(root_bp)
-    app.register_blueprint(surveys_bp)
-    app.register_blueprint(survey_bp)
+
     app.register_blueprint(user_bp)
     app.register_blueprint(users_bp)
     app.register_blueprint(login_bp)
     app.register_blueprint(logout_bp)
     app.register_blueprint(google_bp, url_prefix="/google_oauth")
-    app.register_blueprint(response_bp)
-    app.register_blueprint(observing_system_bp)
-    app.register_blueprint(societal_benefit_area_bp)
-    app.register_blueprint(application_bp)
-    app.register_blueprint(data_product_bp)
-    app.register_blueprint(observing_system_data_product_bp)
-    app.register_blueprint(data_product_application_bp)
-    app.register_blueprint(application_societal_benefit_area_bp)
 
+    # app.register_blueprint(survey_bp)  # TODO:Remove
+    app.register_blueprint(projects_bp)
+    app.register_blueprint(project_bp)
+
+    # Old:
+    # app.register_blueprint(response_bp)
+    # app.register_blueprint(observing_system_bp)
+    # app.register_blueprint(societal_benefit_area_bp)
+    # app.register_blueprint(application_bp)
+    # app.register_blueprint(observing_system_data_product_bp)
+    # app.register_blueprint(data_product_application_bp)
+    # app.register_blueprint(application_societal_benefit_area_bp)
+
+    # TODO: Consider context processors instead?
+    # https://flask.palletsprojects.com/en/2.3.x/templating/#context-processors
     app.jinja_env.globals.update(
         __version__=__version__,
         sqla_inspect=sqla_inspect,
         repo_url=repo.REPO_URL,
         doc_url=repo.DOC_URL,
         discuss_url=repo.DISCUSS_URL,
         current_year=repo.CURRENT_YEAR,
```

Comment:
If I'm going to use this `constants` element, I should probably move many of the above into there as well. Probably should just save that re-organization for later.